catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -132,6 +132,8 @@
 
     private String queryFeedbackEmailDestination;
 
+    private int maximumUploadSize = 1_048_576;
```

Comment:
This is not a product upload max size just a request max size. Meaning the limits on the size of json blobs written by clients. So unless someone uploads a list of metacard ids that is over 1MB in size or the like then this will be fine. Our UI will crash trying to select multiple thousand metacards anyways, so this should be fairly safe